build-study.sh

```diff
@@ -11,11 +11,10 @@ docker_build() {
 
 # used only when rendering stuff for local development.  not used by jenkins
 render_local_config() {
-    local output_dir="${PWD}/${DIR}/output-config"
-    INPUT_DIR=config IMAGE_NAME=$FINAL_IMAGE OUTPUT_DIR=$output_dir STUDY_KEY=$STUDY_KEY STUDY_GUID=$STUDY_GUID ENV=$ENV VERSION=$VERSION MANIFEST=manifest-study.rb OUTPUT_DIR=$output_dir DIR=$output_dir ruby ./configure.rb -y
-    local angular_config_dir="ddp-workspace/projects/${ANGULAR_DIR}/src/assets/config"
-    mkdir -p "${angular_config_dir}"
-    ln -snf "${output_dir}/pepperConfig.js" "${angular_config_dir}/pepperConfig.js"
```

Comment:
This soft-link was causing problems with builds. Updated Angular build config so pepperConfig.js picks up file from output-config